libherokubuildpack/src/inventory/artifact.rs

```diff
@@ -4,6 +4,15 @@ use serde::{Deserialize, Serialize};
 use std::fmt::Display;
 use std::str::FromStr;
 
+/// Representation of a downloadable artifact such as a binary tarball.
+///
+/// An inventory is made up of multiple artifacts that have a version that
+/// can be compared to each other and a URL where the artifact can be downloaded.
+///
+/// Some artifacts may be OS or architectures specific. The checksum value can
+/// be used to validate an artifact once it has been downloaded.
```

Comment:
```suggestion
/// Artifacts are OS and architectures specific. The checksum value can
/// be used to validate an artifact once it has been downloaded.
```

Reply:
The jruby binaries aren't. OS or architecture specific. However, I guess I'm nulling that out manually by writing the same URL twice. So technically your version is true and too. 👍🏻